webpack.config.js

```diff
@@ -123,9 +123,32 @@ var wpconfig = {
                     /studio-frontend/,
                     /paragon/
                 ],
+                exclude: /studio-frontend\/src\/locale_scripts/,
                 use: 'babel-loader'
             },
             {
+                test: /\.json$/,
+                include: /studio-frontend\/i18n\/locales/,
+                use: {
+                    loader: 'file-loader',
+                    options: {
+                        name: '[name].[ext]',
+                        outputPath: 'studio-frontend/locale_messages/'
+                    }
+                }
+            },
+            {
+                test: /\.jsx$/,
+                include: /studio-frontend\/src\/locale_scripts/,
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        name: '[name].[ext]',
+                        outputPath: 'studio-frontend/locale_scripts/'
```

Comment:
I don't think this option is allowed on babel-loader?

At any rate, `locale_scripts/fr.jsx` is not being translated into json and placed somewhere we can find it